examples/events/snowplow-debugger/product.json

```diff
@@ -9,7 +9,8 @@
       },
```

Comment:
The more I am thinking about this, should we also add `minLength` to the `name` field since it is also required?

Reply:
I am not sure why "name" is a required field. I don't think we have any filters related to that in the code today. If we do, we can consider adding minLength otherwise I'd leave it as is

Reply:
I think if we are unsure of the effects then I'll just keep the minLength added to `sku` and we can revisit `name` at a different point. I'm not sure we can be sure of all of the ways adding minLength to `name` will change things 😬